P4-Core-Mechanic/content.md

```diff
@@ -142,14 +142,14 @@ for node:SushiPiece in sushiTower {
   node.runAction(SKAction.moveBy(CGVector(dx: 0, dy: -55), duration: 0.10))
 >
   /* Reduce zPosition to stop zPosition climbing over UI */
-  node.zPosition--
+  node.zPosition -= 1
```

Comment:
Agree the readability is better, although any code snippet changes should be then reflected in the solution.